core-api/src/test/java/com/optimizely/ab/odp/serializer/ODPJsonSerializerTest.java

```diff
@@ -70,9 +70,14 @@ private static ODPEvent createTestEvent(String index) {
         identifiers.put("vuid-" + index + "-2", "fs-" + index + "-2");
         identifiers.put("vuid-" + index + "-3", "fs-" + index + "-3");
 
-        Map<String, String> data = new HashMap<>();
+        Map<String, Object> data = new HashMap<>();
         data.put("source", "java-sdk");
         data.put("data-1", "data-value-" + index);
+        data.put("data-num", Integer.parseInt(index));
+        data.put("data-float", 2.33);
+        data.put("data-bool-true", true);
+        data.put("data-bool-false", false);
```

Comment:
Let's add <null> to this test, which is also supported.

Reply:
I did not get this suggestion. Can you elaborate a bit?

Reply:
"Let's add <nulll> to this test". I missed the key word <null> in my comment :)

Reply:
Testing <null>

Reply:
Testing "null"

Reply:
I should not use "<null>", which is removed by github :) I meant to add a type checking for "null" type.